internal/application/workload.go

```diff
@@ -72,6 +134,10 @@ func (a *Workload) Scale(ctx context.Context, instances int32) error {
 
 // UnbindAll dissolves all bindings from the application.
 func (a *Workload) UnbindAll(ctx context.Context, cluster *kubernetes.Cluster, svcs []string) error {
+
+	// TODO: Optimize this action to perform a single restart for
```

Comment:
Maybe add an issue for this TODO and link it here. Though, 'optimize' issues don't make it in the backlog often...

Reply:
Done, #598